wrangler-storage/src/main/java/io/cdap/wrangler/dataset/schema/HealthcarePackageSchemaLoader.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.wrangler.dataset.schema;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+import io.cdap.wrangler.dataset.schema.SchemaFile.MetaData;
+import io.cdap.wrangler.dataset.schema.SchemaFile.Standard;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * Loads JSON schema from Google Cloud Healthcare API specific system artifact.
+ */
+public class HealthcarePackageSchemaLoader implements JSONSchemaLoader {
```

Comment:
there doesn't appear to be anything healthcare specific here. How is the loader type used?

Reply:
In a follow up PR there will be multiple ways to load DataModels and the type will be used to distinguish the various loaders.